jsdoc/tutorials/02_modelingBasics.md

```diff
@@ -38,9 +38,8 @@ Another valuable way of building shapes is to start with 2D shapes.  These can b
 specifying points, starting with basic primitive 2D shapes, or importing SVG files.  The 2D 
 shapes can then be extruded to produce a wide variety of different geometries.
 ```javascript
-const jscad = require('@jscad/modeling')
-const {polygon } = jscad.primitives
-const { extrudeLinear } = jscad.extrusions
+const { polygon } = require('@jscad/modeling').primitives
```

Comment:
not really a fan of calling require constantly, the old version with `const {polygon } = jscad.primitives` was more to my liking

Reply:
Done. We should also be consistent across documentation and examples as well.